package.json

```diff
@@ -32,28 +45,37 @@
   "homepage": "https://github.com/vasco-santos/ipfs-car#readme",
   "dependencies": {
     "@ipld/car": "^2.1.0",
-    "@types/bl": "^5.0.1",
     "bl": "^5.0.0",
+    "ipfs-core-types": "^0.5.0",
     "ipfs-core-utils": "^0.8.0",
     "ipfs-unixfs-exporter": "https://gitpkg.now.sh/ipfs/js-ipfs-unixfs/packages/ipfs-unixfs-exporter?js-dag-pb-with-dist",
     "ipfs-unixfs-importer": "https://gitpkg.now.sh/ipfs/js-ipfs-unixfs/packages/ipfs-unixfs-importer?js-dag-pb-with-dist",
     "ipfs-utils": "^8.0.0",
     "it-all": "^1.0.5",
     "it-pipe": "^1.1.0",
+    "level": "^7.0.0",
```

Comment:
It is cool, but maybe make the level Blockstore a seperate thing so we don't hae to incldue this dep. I'm thinking of the `npx ipfs-car` use case where you do care if it's quick to download... one motivator here was this module should feel much lighter to use than `js-ipfs` for quick tasks.

Reply:
sounds good, I will make it a new module

Reply:
Done in: https://github.com/vasco-santos/level-blockstore/pull/1

In the future, we can setup a lerna repo with all the content addressable datastores, each one with its own module + interface. This would probably allow us to not be CAR specific. But let's keep things simple for now with the interface in this module until we get to such use cases.